Sources/NIOSSH/Child Channels/ChildChannelStateMachine.swift

```diff
@@ -24,8 +24,8 @@ struct ChildChannelStateMachine {
     }
 }
 
-extension ChildChannelStateMachine {
-    fileprivate enum State: Hashable {
+private extension ChildChannelStateMachine {
```

Comment:
NIO style is to place the access modifiers within the extensions, not on the extension itself.